workflows/dfe.snake

```diff
@@ -249,19 +249,18 @@ rule download_dfe_alpha:
     output:
         "ext/dfe-alpha-release-2.16/est_dfe",
         "ext/dfe-alpha-release-2.16/data/n1_100/s_evaluated.dat"
-
     shell:
         """
         cd ext
-        wget -c https://sourceforge.net/projects/dfe-alpha-k-e-w/files/dfe-alpha-release-2.16.tar.gz/download
-        mv download dfe-alpha-release-2.16.tar.gz
-        tar -xvf dfe-alpha-release-2.16.tar.gz
-        cat dfe_alpha_makefile_stdpopsim_patch > dfe-alpha-release-2.16/Makefile && cd dfe-alpha-release-2.16 && make
+        mkdir -p dfe-alpha-release-2.16/
+        cd dfe-alpha-release-2.16/
+        wget -O est_dfe http://sesame.uoregon.edu/~stittes/stdpopsim/est_dfe
+        chmod u+x est_dfe
         wget http://sesame.uoregon.edu/~adkern/stdpopsim/data.tar.gz && tar -xvf data.tar.gz && rm -f data.tar.gz
         # old location of above
         #  https://datashare.ed.ac.uk/bitstream/handle/10283/2730/data.tar.gz?sequence=1&isAllowed=y
         cd ../
-        rm dfe-alpha-release-2.16.tar.gz
+        rm -f dfe-alpha-release-2.16.tar.gz
```

Comment:
I suppose we could drop this since we aren't actually downloading it with this PR. The `-f` ensures it works even if the file isn't there.